pom.xml

```diff
@@ -59,6 +59,8 @@
         <dep.jmxutils.version>1.21</dep.jmxutils.version>
```

Comment:
What is your plan for testing?

Reply:
Im looking to this: https://www.testcontainers.org/modules/databases/oraclexe/

putting a container with a oracle database and test against it..
but Im getting trouble with docker image.. looks like it needs a login to download oracle docker

on a small test Im testing this with a real oracle database, that way i now the issues with NUMERIC and CLOB

Reply:
maybe I could use H2 with Oracle Compatibility Mode if this it not a issue

Reply:
FYI there is `wnameless/oracle-xe-11g-r2` that can be used for testing and the DMCA issue has been resolved

Reply:
version 11g has some limitations on naming (no more than 30 chars)
as presto first create a tmp table like this TMP_PRESTO_5BF7F7B182614ED58983CA9E8881023A this will fail.

testing on a 12g (local database, not found a docker image)
I cant create a table but it fails with 
ORA-14047: ALTER TABLE|INDEX RENAME may not be combined with other operations

debuging i think that first we create a TMP table and then rename to the final.. 
this on Oracle its not possible..
ALTER TABLE "TEST"."TMP_PRESTO_5BF7F7B182614ED58983CA9E8881023A" RENAME TO "TEST"."TEST_TABLE"

I manage to do some select test on a previous create table..

do you think that if I create the tables with OracleDriver and test only the selects its valid?

Reply:
updated with a small test..
if this is valid I could make some more tests